### PR TITLE
Add subject filter. Filter out non-cardiovascular related documents.

### PR DESCRIPTION
**Pull request by thomascpan**

### CODE REVIEW

**Review by carolynyen — COMMENTED** [review]
looks good, tho I'm just wondering about a few things. 
Is the global constant name CARDIOVASCULAR descriptive enough? Also, is it good practice to have it global?
Also, in subject_filter, is there a difference in simply iterating and checking if the keywords are in the cardiovascular terms set?

File: processes/dags/pubmed_migration.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 # Setting up MongoDB hook to mlab server
 mongodb_hook = MongoHook('mongo_default')
 ftp_conn_id = "pubmed_ftp"
+CARDIOVASCULAR = "blood/heart and lymphatics"
```
Comment (carolynyen) [inline processes/dags/pubmed_migration.py:25]: Does it work to have blood/heart, or should those be listed separately?

Comment (thomascpan) [reply processes/dags/pubmed_migration.py:25]: i basically gathered all subjects i found in a subset and this was the closest thing to cardiovascular I found. I'm not exactly sure how they pick their subject names, but this was  found in a significant chunk of them. I might redo this later and try and make it more sophisticated, but i just wanted to get something out for joey first.

**Comment by thomascpan** [comment]
Maybe a global CONSTANT isn't the best, but some of our other variables are also global right now. Not sure what's the alternative right now. 
I thought it would be more concise to write it this way than have a nested loop.

**Pull request merged**